File: admin/src/types/careerDepartment.ts
export interface CareerDepartment {
  departmentId: number;
  name: string;
  displayOrder: number;
  isActive: boolean;
  createdDate?: string;
  createdBy?: string;
  modifiedDate?: string;
  modifiedBy?: string;
}

export interface CareerDepartmentRequest {
  name: string;
  displayOrder: number;
  isActive: boolean;
}

File: backend/src/MalihaPolytex.Domain/Entities/CareerDepartment.cs
using System;

namespace Backend.Entities;

public class CareerDepartment
{
    public int DepartmentId { get; set; }

    public string Name { get; set; } = string.Empty;

    public int DisplayOrder { get; set; }

    public bool IsActive { get; set; }

    public DateTime CreatedDate { get; set; }

    public string? CreatedBy { get; set; }

    public DateTime? ModifiedDate { get; set; }

    public string? ModifiedBy { get; set; }
}